go.mod

```diff
@@ -78,7 +78,7 @@ require (
 	github.com/hashicorp/vault-plugin-secrets-ad v0.6.6
 	github.com/hashicorp/vault-plugin-secrets-alicloud v0.5.5
 	github.com/hashicorp/vault-plugin-secrets-azure v0.6.2-0.20200818154643-63f769e87b12
-	github.com/hashicorp/vault-plugin-secrets-gcp v0.6.4-0.20200727183837-4005bee700bf
```

Comment:
Why is the GCP dep changing?

Reply:
I have no idea!

Reply:
Ah, I had something committed on the release branch without realizing it. Should be good to go now.